frontend/src/main/web/src/components/GroupTeaser.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react' // eslint-disable-line
+import Link from '../components/Link'
+import View from '../components/View'
+import Icon from '../components/Icon'
+
+const viewTheme = {
+  base: {
+    ai: 'Ai(c)',
+    fld: '',
+    m: 'Mb(rh)'
+  }
+}
+
+const statusIcons = {
+  ACTIVE: '',
+  READONLY: 'locked',
+  OBSOLETE: 'trash'
+}
+
+const GroupTeaser = ({
+  details,
+  name,
+  children,
+  ...props
```

Comment:
No `propTypes` are specified below. What are the types of these props and what do they do?